conda-store-server/conda_store_server/_internal/schema.py

```diff
@@ -23,12 +23,15 @@ def _datetime_factory(offset: datetime.timedelta):
 # this is a more restrictive list
 ALLOWED_CHARACTERS = "A-Za-z0-9-+_@$&?^~.="
 ARN_ALLOWED = f"^([{ALLOWED_CHARACTERS}*]+)/([{ALLOWED_CHARACTERS}*]+)$"
+ARN_ALLOWED_REGEX = re.compile(ARN_ALLOWED)
```

Comment:
What does ARN stand for? seems to be either `name` `namespace` or a combination of these?

Reply:
perhaps best to add a comment is needed

Reply:
I don't know what ARN stands for, but it seems to mean "a key in a RoleMapping". It's the part of a role mapping that describes what namespace/environment combination a role is mapped to.